integration-tests/ssh/pom.xml

```diff
@@ -51,6 +59,11 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
```

Comment:
Do we need this?

Reply:
I think I used the assertion for asserj-core , but surely not necessary

Reply:
removed